src/solvers/dgsem_p4est/dg.jl

```diff
@@ -91,4 +91,6 @@ include("dg_3d_parabolic.jl")
 include("dg_parallel.jl")
 
 include("subcell_limiters_2d.jl")
+include("subcell_limiters_3d.jl")
+include("dg_3d_subcell_limiters.jl")
```

Comment:
Maybe one more thing: Wouldn't it be more consistent if the `3d` gets placed at last, i.e., 
`dg_subcell_limiters_3d.jl`?